pkg/envoy/ads/server.go

```diff
@@ -32,6 +32,9 @@ const (
 
 	// workerPoolSize is the default number of workerpool workers (0 is GOMAXPROCS)
 	workerPoolSize = 0
+
+	// xdsServerCertificateCommonName is the common name of the certificate for the ADS server
+	xdsServerCertificateCommonName = "ads"
```

Comment:
Should the ads cert cn also contain the namespace of the control plane?

Reply:
I would think so

Reply:
Will address this in a separate issue/PR